kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -679,9 +783,6 @@ class ResidualBasedBlockBuilderAndSolver
 
         KRATOS_INFO_IF("ResidualBasedBlockBuilderAndSolver", ( this->GetEchoLevel() > 2 && rModelPart.GetCommunicator().MyPID() == 0)) << "Finished setting up the dofs" << std::endl;
 
-        KRATOS_INFO_IF("ResidualBasedBlockBuilderAndSolver", ( this->GetEchoLevel() > 2)) << "End of setup dof set\n" << std::endl;

```

Comment:
can you please make a separate PR for changes unrelated to this one?

Reply:
I could ... but this is doing exactly the same as in line 680 just above except that line 680 already has a correction for MPI which is not present here. So... i would make an exception for this specifically.

Reply:
Also i think this is a merge feature: @roigcarlo corrected the line above, and the merge did not recognize that the two lines should one replace the other

Reply:
ok